test/box-luatest/gh_9954_mvcc_with_exclude_null_test.lua

```diff
@@ -107,3 +107,53 @@ g.test_mvcc_with_exclude_null_space_drop = function()
         box.space.test:drop()
     end)
 end
+
+-- gh-10396
```

Comment:
Nit: up to you, but we usually create separate files for each issue nowadays.

Reply:
Yep, but this file is dedicated to mvcc with `exclude_null`, so I decided that it belongs here. I would say, it's actually follow up of `9954`.